### DISCUSSION TITLE
Qodana scan message cannot find SDK

### DISCUSSION THREAD

**Discussion by huo2huo** [JetBrains/Qodana#85] [Q&A]
What should I do？
![sdk](https://user-images.githubusercontent.com/101409437/165707709-2430f448-b33e-47c3-87c3-725b2c940b02.png)

**Comment by brichbash** [comment]
As far as I understand, this thread relates to the [thread №84](https://github.com/JetBrains/Qodana/discussions/84).
Could you please share your `.idea/misc.xml` after the failure?

**Reply by huo2huo** [reply]
this is my failed project of misc.xml.
`<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="FrameworkDetectionExcludesConfiguration">
    <file type="web" url="file://$PROJECT_DIR$/code/pc-web/pc-web-boot" />
  </component>
  <component name="MavenProjectsManager">
    <option name="originalFiles">
      <list>
        <option value="$PROJECT_DIR$/code/pom.xml" />
      </list>
    </option>
  </component>
</project>`

this is my successful project of misc.xml.
`<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="FrameworkDetectionExcludesConfiguration">
    <file type="web" url="file://$PROJECT_DIR$/code/pc-web/pc-web-boot" />
  </component>
  <component name="MavenProjectsManager">
    <option name="originalFiles">
      <list>
        <option value="$PROJECT_DIR$/code/pom.xml" />
      </list>
    </option>
  </component>
  <component name="ProjectRootManager" version="2" languageLevel="JDK_14" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK" />
</project>`

**Reply by brichbash** [reply]
`.idea` is auto-generated and used during Qodana run (when opening in Intellij IDEs as well), so if a project is complicated then Qodana may fail to properly initialize it. I suggest you use `misc.xml` that you got after manually setting up the project using Intellij IDEA if that works for you.

**Reply by huo2huo** [reply]
good，thanks.Whether the projects under Linux need to download a Linux version of idea, then configure the project and generate Idea folder.
